cluster/repository/cluster.go

```diff
@@ -98,6 +111,16 @@ func (m *GormClusterRepository) Load(ctx context.Context, id uuid.UUID) (*Cluste
 	return &native, errs.WithStack(err)
 }
 
+func (m *GormClusterRepository) LoadClusterByURL(ctx context.Context, url string) (*Cluster, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "cluster", "loadClusterByURL"}, time.Now())
+	var native Cluster
+	err := m.db.Table(m.TableName()).Where("url = ?", url).Find(&native).Error
+	if err == gorm.ErrRecordNotFound {
+		return nil, err
```

Comment:
Lets use NotFoundError for consistency - https://github.com/fabric8-services/fabric8-cluster/blob/master/cluster/repository/identity_cluster.go#L65-L67

Reply:
done in 191866f